src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -30,19 +30,14 @@
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
       
-      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
-      <RunTimeArtifactsExcludeFiles Include="linuxonjit.dll" />
-      
       <RunTimeArtifactsIncludeFolders Include="/" />
       <RunTimeArtifactsIncludeFolders Include="IL/" />
       <RunTimeArtifactsIncludeFolders Include="crossgen2/" />
-      <RunTimeArtifactsIncludeFolders Include="PDB/" />
```

Comment:
The PDB folder is how we get stack traces for runtime crashes. We shouldn’t remove it (or we should go straight from including the folder wholesale to including specific files in the folder)

Reply:
Hmm, can you please explain to me how Helix test executions locate the folder?

Reply:
I don’t remember exactly how, but we pass information somewhere in an environment variable so Watson can find the PDBs.

Reply:
> Hmm, can you please explain to me how Helix test executions locate the folder?

It here https://github.com/dotnet/runtime/blob/d68342ceeb69adff35d5b3462091d339a700c1a5/src/coreclr/tests/helixpublishwitharcade.proj#L196

Reply:
Thanks Jeremy and Egor for your feedback. I have put back the PDB folder and just added explicit exclusions for files that don't seem related to runtime call stacks. Please let me know if you believe any of the files I'm removing need to be kept. This still reduces the folder by about 175 MB.